src/admin/menu/views/MenuOverview.tsx

```diff
@@ -48,7 +51,26 @@ const MenuOverview: FunctionComponent<MenuOverviewProps> = ({ history }) => {
 	};
 
 	const renderMenuOverview = (data: Partial<Avo.Menu.Menu>[]) => {
-		return (
+		if (data.length) {
```

Comment:
Is dit geen logischere werkwijze. Indien je geen data hebt, moet je de menu ook niet setten, right? Dan toont de applicatie gewoon de error.

```
if (!data.length) {
	return (
		<ErrorView message="Er zijn nog geen navigaties aangemaakt.">
			<p>
				Lorem ipsum dolor sit amet consectetur adipisicing elit. Maiores aliquid ab debitis
				blanditiis vitae molestiae delectus earum asperiores mollitia, minima laborum expedita
				ratione quas impedit repudiandae nisi corrupti quis eaque!
			</p>
			<Spacer margin="top">
				<Button
					icon="plus"
					label="Navigatie toevoegen"
					onClick={() => history.push(MENU_PATH.MENU_CREATE)}
				/>
			</Spacer>
		< /ErrorView>
	);
}

setMenus(data);

return (
	<Table
		columns={MENU_OVERVIEW_TABLE_COLS}
		data={data}
		renderCell={(rowData: Partial<Avo.Menu.Menu>, columnId: string) =>
			renderTableCell(rowData, columnId as MenuOverviewTableCols)
		}
		rowKey="id"
		variant="bordered"
	/>
);
```